Frontend/ui-driver/src/Helpers/Utils.js

```diff
@@ -555,3 +555,9 @@ export const renderSlider = function (cb) {
     }
   }
 }
+
+  // This can be used in future for activityResultListeners
+  // window.activityResultListeners[requestCode] = function (reqCode, bundle) {
+  //   cb(reqCode)(bundle)(); 
+  // };
+}
```

Comment:
we can remove this extra `}`, because this will break the compilation in foreign.js